examples/aks.preflight.yaml

```diff
@@ -16,9 +16,9 @@ enabled-packages:
 
 # This configures how the results will be reported.
 outputs:
-# writes a human readable doc that can be opened with a browser.
-- type: local
-  path: ./output
+# uploads a json file to Azure Blob Storage
```

Comment:
Could you also add a note here that the environment variables have to be set, and link to somewhere that explains how to do that?

Reply:
Added to `./docs/configuration.md`.